spec/support/fixtures/openlibrary/response_success.json

```diff
@@ -0,0 +1,67 @@
+{
+  "ISBN:0385472579": {
+    "publishers": [
+      {
+        "name": "Anchor Books"
+      }
+    ],
+    "pagination": "159 p. :",
+    "identifiers": {
+      "lccn": [
+        "93005405"
+      ],
+      "openlibrary": [
+        "OL1397864M"
+      ],
+      "isbn_10": [
+        "0385472579"
+      ],
+      "goodreads": [
+        "979250"
+      ],
+      "librarything": [
+        "192819"
+      ]
+    },
+    "subtitle": "shouts of nothingness",
+    "title": "Zen speaks",
+    "url": "https://openlibrary.org/books/OL1397864M/Zen_speaks",
+    "number_of_pages": 159,
+    "cover": {
+      "small": "https://covers.openlibrary.org/b/id/240726-S.jpg",
+      "large": "https://covers.openlibrary.org/b/id/240726-L.jpg",
+      "medium": "https://covers.openlibrary.org/b/id/240726-M.jpg"
+    },
+    "subjects": [
+      {
+        "url": "https://openlibrary.org/subjects/caricatures_and_cartoons",
+        "name": "Caricatures and cartoons"
+      },
+      {
+        "url": "https://openlibrary.org/subjects/zen_buddhism",
+        "name": "Zen Buddhism"
+      }
+    ],
+    "publish_date": "1994",
+    "key": "/books/OL1397864M",
+    "authors": [
+      {
+        "url": "https://openlibrary.org/authors/OL223368A/Zhizhong_Cai",
+        "name": "Zhizhong Cai"
+      }
+    ],
+    "classifications": {
+      "dewey_decimal_class": [
+        "294.3/927"
+      ],
+      "lc_classifications": [
+        "BQ9265.6 .T7313 1994"
+      ]
+    },
+    "publish_places": [
+      {
+        "name": "New York"
+      }
+    ]
+  }
+}
```

Comment:
Add empty line at end of file. If you are using vscode you can add this settings 
`  "files.insertFinalNewline": true`